core-bundle/src/Crawl/Escargot/Subscriber/BrokenLinkCheckerSubscriber.php

```diff
@@ -93,7 +92,7 @@ public function needsContent(CrawlUri $crawlUri, ResponseInterface $response, Ch
         ++$this->stats['ok'];
 
         // Skip any redirected URLs that are now outside our base hosts (#4213)
-        $actualHost = (new Uri($response->getInfo('url')))->getHost();
+        $actualHost = parse_url($response->getInfo('url'), PHP_URL_HOST);
```

Comment:
Is `$response->getInfo('url')` guaranteed to have a lowercase host name? Otherwise we cannot use `parse_url()` here I think.

Reply:
Yes, but the very fact that we have to think about it is a telltale sign that we should always use the library. 😄